draft-ietf-taps-interface.md

```diff
@@ -1647,7 +1647,9 @@ to the new Connection when calling Clone(), but in this case, a later change to
 `Connection Priority` on one Connection does not change it on the
 other Connections in the same Connection Group.
 
-Message Properties are also not entangled.  For example,
+Message Properties set on a Connection are also not entangled. Chaning these on an individual Connection
+which is part of a Connections Group SHOULD have the same effect as setting the Message property on each
+Message sent though that Connection. For example,
```

Comment:
The sentence that you included here is only making things more confusing, IMO. Suggest to remove it.

Reply:
I think a first modified sentence is sufficient "Message Properties set on a Connection also apply only to that Connection". Actually reading it again now I realize that it is the example that confuses things, because that talks about setting a Message Property on a Message, so maybe we can just remove the example as it does not belong in this section.

Reply:
applied